framework/log/Target.php

```diff
@@ -189,6 +189,12 @@ public function setLevels($levels)
                 }
             }
         } else {
+            $bitmapValues = array_reduce($levelMap, function ($carry, $item) {
+                return $carry | $item;
+            });
+            if (!($bitmapValues & $levels) && $levels != 0) {
+                throw new InvalidConfigException("Incorrect bitmap value : $levels");
```

Comment:
I think it's better to say `Incorrect $levels value.` according to method signature.